### PR TITLE
Model zoo style

### PR DESCRIPTION
**Pull request by kevinthesun**
Fix model zoo related style issues.

### CODE REVIEW

File: docs/model_zoo/index.md [hunk 1]
```diff
@@ -4,89 +4,201 @@ MXNet features fast implementations of most state-of-the-art models reported in
...
+* [CIFAR10](https://www.cs.toronto.edu/~kriz/cifar.html): 60,000 natural images (32 x 32 pixels) from 10 categories.
+* [PASCAL_VOC](http://host.robots.ox.ac.uk/pascal/VOC/): A subset of ImageNet images with object bounding boxes.
+* [UCF101](http://crcv.ucf.edu/data/UCF101.php): 13,320 videos from 101 action categories.
+* [Mini-Places2](http://6.869.csail.mit.edu/fa15/project.html): Subset of the Places2 dataset. Includes 100,000 images from 100 scene categories.
+* ImageNet 11k
+* [Places2](http://places2.csail.mit.edu/download.html): There are 1.6 million train images from 365 scene categories in the Places365-Standard, which are used to train the Places365 CNNs. There are 50 images per category in the validation set and 900 images per category in the testing set. Compared to the train set of Places365-Standard, the train set of Places365-Challenge has 6.2 million extra images, leading to totally 8 million train images for the Places365 challenge 2016. The validation set and testing set are the same as the Places365-Standard.
+
+
+
+```eval_rst
```
Comment (piiswrong) [inline docs/model_zoo/index.md]: this doesn't seem to be showing correctly

Comment (kevinthesun) [inline docs/model_zoo/index.md]: You mean the page doesn't show correctly here?

Comment (piiswrong) [inline docs/model_zoo/index.md]: here https://github.com/kevinthesun/mxnet/blob/639c5eda5dee269c0b3b47d6c8b1a33f324c4a4f/docs/model_zoo/index.md

**Comment by mli** [comment]
@kevinthesun i wrote a script that first converts a markdown table into the rst format when `make html`. it modifies the .md files directly. so please don't commit the new generated .md files which contain '```rst' after running `make html`.

File: docs/model_zoo/index.md (hunk 1 above)
Comment (kevinthesun) [inline docs/model_zoo/index.md]: After compilation, the tables of this md file are automatically converted to eval_rst. I'll keep this file as original and push again.

**Comment by mli** [comment]
lgtm

**Pull request merged**